setup.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
```

Comment:
And `#!/usr/bin/env python` will work on unix systems, so mac and linux. I'm not sure if windows will work however.